index.html

```diff
@@ -46,92 +46,8 @@ <h1>Abdel-Khafid SALAOU</h1>
-        <div class="project-card-info deskno">
-            <h3>
-                Hydraulic head loss calculator online
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card three">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_3.svg" alt="My third project"></a>
-        <div class="project-card-info deskno">
-            <h3>
-                Online scientific calculator
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card four">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_4.svg" alt="My fourth project"></a>
-        <div class="project-card-info deskno">
-            <h3>
-                Professional Portfolio
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card deskreduce">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_5.svg" alt="My fifth project"></a>
-        <div class="project-card-info">
-            <h3>
-                Currency converter
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card deskno">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_6.svg" alt="My sixth project"></a>
-        <div class="project-card-info">
-            <h3>
-                Visit counter
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
    </section>
+   <section class="pop-window"></section>
```

Comment:
- [  ] As suggested by the previous reviewer, kindly add the colour font on the button, it's still as it was on the last review. The design was made to give a better understanding from a user perspective, kindly fix this 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/224492739-a58fca88-6a93-4fe1-ae21-393a1e0a50aa.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/224492769-d2200324-0de2-46f2-8615-3617bdd15be1.png)

Reply:
Hello @Shedrack-Sunday, thank you for your time. I didn't get the last part. Can you please give me more details?
In the Figma template, the background is white and the text color is #1A2236. 
The following picture represents what I did before the first review and I changed it to match the Figma template.
![cap1](https://user-images.githubusercontent.com/119133987/224493399-a844ca5b-5b06-4af3-b3c1-25e440da833f.PNG)